src/main/java/com/microsoft/azure/spring/data/cosmosdb/common/TelemetryEventTracker.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.azure.spring.data.cosmosdb.common;
+
+import com.microsoft.applicationinsights.TelemetryClient;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NonNull;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class TelemetryEventTracker {
+
+    private static final String PROPERTY_INSTALLATION_ID = "installationId";
+
+    private static final String PROPERTY_VERSION = "version";
+
+    private static final String PROPERTY_SERVICE_NAME = "serviceName";
+
+    private static final String PROJECT_INFO = "spring-data-cosmosdb/" + PropertyLoader.getProjectVersion();
+
+    @Getter(AccessLevel.PRIVATE)
```

Comment:
Why private Getter is needed here and we are not accessing the variable directly?

Reply:
Just personal style prefer here.